yaml/xyz/openbmc_project/Provisioning/Provisioning.interface.yaml

```diff
@@ -0,0 +1,28 @@
+description: >
+    Interface to represent the provisioning status of the BMC. Provides a
+    property to indicate whether the BMC is provisioned, and methods to initiate
+    provisioning and check the peer BMC connection.
+properties:
+    - name: ProvisioningState
+      type: boolean
+      flags:
+          - readonly
+      default: false
+      description: >
+          True means the BMC is in a provisioned state.
+
+methods:
+    - name: StartProvisioning
+      description: >
+          Starts the provisioning process and updates the ProvisioningState
+          accordingly.
+
+    - name: CheckPeerBMCConnection
+      description: >
+          Performs a check to determine if the peer BMC is reachable and and if
+          already provisioned.
```

Comment:
> if already provisioned.

How do we check the peer BMC’s provision state? Are we able to read the peer BMC’s pic-control, or did you mean something else? My understanding is that Provisioned indicates _this BMC_ is provisioned, but does it also imply verification? By verification, I mean the mTLS handshake performed through this API (by checking the Provisioned property.) after certificate exchange.